pkg/scyllaclient/config_client.go

```diff
@@ -197,6 +197,18 @@ func (c *ConfigClient) AlternatorHTTPSPort(ctx context.Context) (string, error)
 	return fmt.Sprint(resp.Payload), err
 }
 
+// UUIDSStableIdentifiers returns node alternator HTTPS port.
+func (c *ConfigClient) UUIDSStableIdentifiers(ctx context.Context) (bool, error) {
+	resp, err := c.client.Config.FindConfigUUIDSstableIdentifiersEnabled(config.NewFindConfigUUIDSstableIdentifiersEnabledParamsWithContext(ctx))
+	if alternatorDisabled(err) {
+		return false, nil
+	}
```

Comment:
But still, why would we check for `alternatorDisabled(err)` here?
I know that it is just a cover for checking status 400, but maybe it would be better to check for that in a more direct way.

Reply:
Could you please elaborate

Reply:
I meant that checking for status 400 by using `alternatorDisabled(err)` looks strange. Now it looks good:)

Reply:
Oh, yes, I have renamed it.